packages/playwright/src/index.ts

```diff
@@ -53,7 +53,7 @@ export const devices = {
   },
 };
 
-export function createReplayReporterConfig(config: ReplayPlaywrightConfig) {
+export function createReplayReporterConfig(config: ReplayPlaywrightConfig = {}) {
```

Comment:
Didn't you rename this fella

Reply:
Yeah, I did - I have to rebase this PR in the morning 😉